testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js

```diff
@@ -869,70 +925,67 @@ QUnit.module('Scenarios', moduleConfig, () => {
         });
     });
 
-    QUnit.test('Export [string(A,B) x None x number]', function(assert) {
+    QUnit.test('Export [string(A,B) x None x None] & showRowGrandTotals', function(assert) {
         const done = assert.async();
         const ds = {
             fields: [
                 { area: 'row', dataField: 'row1' },
-                { area: 'data', summaryType: 'count', dataType: 'number' }
             ],
             store: [
-                { row1: 'A' },
-                { row1: 'B' },
-                { row1: 'B' }
+                { row1: 'A' }, { row1: 'B' }
             ]
         };
 
         const pivotGrid = $('#pivotGrid').dxPivotGrid({
             showColumnGrandTotals: false,
-            showRowGrandTotals: false,
+            showRowGrandTotals: true,
```

Comment:
again significant change in config: the old config is not tested with new changes